lib/src/room.dart

```diff
@@ -612,14 +615,42 @@ class KurentoRoom extends ChangeNotifier implements Room {
           await _updateServiceRequestStatus('ASSIGNED', agentFirstName);
         } else if (_serviceRequestState == ServiceRequestState.assigned) {
           _serviceRequestState = ServiceRequestState.started;
-          notifyListeners();
 
           // Once we've started, we no longer need the timer to consume resources. This means we may miss an
           // Agent-initiated end message, but the impact of that is low (the Explorer can end the call themselves).
           _getServiceRequestStatusTimer?.cancel();
 
+          // HACK: This condition is here to cover the improbable possibility
+          // that we would have join the room with a _localStream which didn't
+          // yet contain a video track. In this case, this code would replace
+          // the transceiver for another one containing the video track.
+          final bool shouldTransmitVideo = _isVideoMuted == false && _hasVideoTrack;
+          final bool hasOutgoingVideoSenderTrack = _connectionByTrackId[_localTrackId!]?.ownsVideoTrack == true;
+          if (null != _localTrackId && shouldTransmitVideo && !hasOutgoingVideoSenderTrack) {
```

Comment:
If `_localTrackId` was `null`, wouldn't the previous line (`_localTrackId!`) blow up?